pages/expense.js

```diff
@@ -35,9 +39,12 @@ class ExpensePage extends React.Component {
   }
 
   render() {
-    const { data, ExpenseId, LoggedInUser } = this.props;
+    const { data, ExpenseId, LoggedInUser, createSuccess } = this.props;
+    const expenseCreated = {
```

Comment:
What's the intent to create this object rather than using `id` directly?

Reply:
The `ExpenseWithData` component used the same variable name `ExpenseId` for its `id` attribute